src/main/java/org/frankframework/frankdoc/util/XmlBuilder.java

```diff
@@ -46,15 +46,24 @@
 
 	private Element element;
 
+	// Only for toString()
+	private final String tagName;
+	private String valueOfAttributeName = "<no name>";
```

Comment:
Deze aanpassingen moet je er weer uit halen

Reply:
Gaat gebeuren in https://github.com/ibissource/frank-doc/pull/103.